js/ccf-app/src/polyfill.ts

```diff
@@ -52,13 +52,20 @@ class KvMapPolyfill implements KvMap {
   delete(key: ArrayBuffer): boolean {
     return this.map.delete(base64(key));
   }
+  clear(): undefined {
+    this.map.clear();
+    return undefined;
+  }
   forEach(
     callback: (value: ArrayBuffer, key: ArrayBuffer, kvmap: KvMap) => void
   ): void {
     this.map.forEach((value, key, _) => {
       callback(value, unbase64(key), this);
     });
   }
+  get size(): number {
+    return this.map.size;
+  }
```

Comment:
Questions for @letmaik:

- JS says that `Map.clear` [returns undefined](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Map/clear#return_value), but the TS `Map` returns void. I think we want to match the documented JS API, but maybe I'm missing something.
- Is this a sane way to implement this property getter? I was hoping I could just write an alias like `readonly size = this.map.size;`, but it seems the `readonly` that's fine in the `interface` means something very different (const, assigned-on-construction) in the implementations.

Reply:
Regarding the second point, the getters look fine. I don't think there's a substantially nicer way to do this.